### PR TITLE
Jr comparison validate build once

### PR DESCRIPTION
**Pull request by smuzaffar**

### CODE REVIEW

**Comment by smuzaffar** [comment]
please test

File: comparisons/validateJR.sh [hunk 1]
```diff
@@ -21,20 +21,23 @@ ls -d ${baseA}/[1-9]* | while read -r d; do
...
     fN=`echo ${baseA}/${fNP} | cut -d" " -f1 | sed -e "s?^${baseA}/??g"`
     #[ ! -f "${baseA}/${fN}" ] && echo Missing ${baseA}/${fN}
     #process regular files first; need files both in baseA and baseB
     if [ -f "${baseA}/${fN}" -a -f "${baseB}/${fN}" ]; then
-	extN=all_${diffN}_${dsN}
-	mkdir -p ${extN}
-	pushd ${extN}
-	  cp ${VALIDATE_C_SCRIPT} ./
-	  echo "Will run on ${fN} in ${extN}"
-	  g++ -shared -o validate.so validate.C `root-config --cflags ` -fPIC
```
Comment (slava77) [inline comparisons/validateJR.sh:35]: just for posterity, this part was removed in local test setups in 2015 https://github.com/slava77/cms-reco-tools/commit/8129c94139793bd111359e64badeb857ef364c1e
I guess it never fully propagated to this repo

**Comment by smuzaffar** [comment]
please test

**Comment by smuzaffar** [comment]
please test

**Comment by smuzaffar** [comment]
`validateJR.sh` now runs much faster (takes 10mins instead of 25 to 30).

**Pull request merged**